packages/web-components/fast-components/src/tooltip/fixtures/base.html

```diff
@@ -0,0 +1,151 @@
+<fast-design-system-provider use-defaults>
+    <h1>Tooltip</h1>
+
+    <h4>Default</h4>
```

Comment:
Our headings should be semantic for those using screen readers.  They seem to be used stylistically when they should be used contextually and semantically.  So H4 really should be H2.  Additionally, should we be attributing with styles?  As documentation, should we be using our own design system just like a production scenario would?

Reply:
Can we create an issue to do a larger pass here? This is an issue across all controls and not specific to this one, outside that I agree - this is net new and should be correct.

Reply:
https://github.com/microsoft/fast/issues/3812